test_aiodataloader.py

```diff
@@ -357,8 +368,12 @@ def do_resolve(x):
 
 async def test_can_call_a_loader_from_a_loader():
     deep_loader, deep_load_calls = id_loader()
-    a_loader, a_load_calls = id_loader(resolve=lambda keys:deep_loader.load(tuple(keys)))
-    b_loader, b_load_calls = id_loader(resolve=lambda keys:deep_loader.load(tuple(keys)))
+
+    async def do_resolve(keys):
+        return await deep_loader.load(tuple(keys))
+
+    a_loader, a_load_calls = id_loader(resolve=do_resolve)
+    b_loader, b_load_calls = id_loader(resolve=do_resolve)
```

Comment:
Why change this?

Reply:
Making it a lambda caused it to fail `resolve`'s type check because they're not returning `Awaitable`s